src/assets/json/checklist/tests-web-checklist-en.json

```diff
@@ -0,0 +1,155 @@
+	"verifier": [["Everything that can be done with the mouse can be done using the keyboard and vice-versa"],["The focus movement order maintains a logical reading and usage flow."],["The focus does not get stuck on a part of the page (keyboard trap)."],["The focus indicator (outline) is visible on all interactive elements and has sufficient contrast."]],
+	"resultat": ["All interactive elements can be reached from a keyboard navigation", "All interactive elements are usable with keyboard interactions"],
+	"exception": "",
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/keyboard-navigation/#allow-the-use-of-the-main-features-of-the-application-with-the-keyboard",
+	"profils": "",
+	"wcag": ["2.1.1"],
+	"ID": "testWebID-38",
+	"IDorigin": "testWebID-38"
+}, {
+	"themes": "Layout",
+	"title": "Use relative sizes and responsive design",
+	"type": ["Manual test"],
+	"tests": ["With Firefox, from the 'View' menu, select 'Zoom' and then 'Zoom Text Only' and set the zoom level to 200%."],
+	"verifier": ["There is no truncated or hidden content and no unusable features"],
+	"resultat": ["No truncated or hidden contents and no unusable features"],
+	"exception": "",
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/layout/#use-relative-size-to-allow-changing-text-size-and-responsive-design",
+	"profils": "",
+	"wcag": ["1.4.11"],
+	"ID": "testWebID-42",
+	"IDorigin": "testWebID-42"
+}, {
+	"themes": "Forms",
+	"title": "Make sure an accessible name is associated with each form field",
+	"type": ["Manual test - Code inspector"],
+	"tests": ["Run the code inspector of the browser on the 'Accessibility' tab."],
+	"verifier": ["Each form field must have an accessible name."],
+	"resultat": ["Each field has at least one relevant accessible name and contains at least the text of the visible field label on the screen (a placeholder is not compliant)."],
+	"exception": "The accessible name can be visually hidden when the role of a form field is clear and unambiguous (for example a search field accompanied by a magnifying glass button, a checkbox to select a row in a data table).",
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/forms/#make-form-fields-accessible",
+	"profils": "",
+	"wcag": ["2.4.6", "3.3.2"],
+	"ID": "testWebID-47",
+	"IDorigin": "testWebID-47"
+}, {
+	"themes": "Forms",
+	"title": "Make sure error messages are relevant",
+	"type": ["Manual test"],
+	"tests": ["Fill out the forms with incorrect data and leave required fields empty.","Submit the forms."],
+	"verifier": ["Error messages are present, relevant, and identify the fields with errors."],
+	"resultat": ["Errors are detected and messages relevant"],
+	"exception": [],
```

Comment:
No exception so I think you can remove it from the file to prevent an empty section in the page: 
![screenshot-deploy-preview-603--a11y-guidelines_netlify_app-2025_01_07-08_11_10](https://github.com/user-attachments/assets/64d2fb31-baca-4ba8-855b-9505803d5929)